pheps/phep-0001.md

```diff
@@ -0,0 +1,287 @@
+
+    In a way this section can be thought of as a breakout section of the Rationale section that is focused specifically on why certain ideas were not ultimately pursued.
+
+11. Open Issues - While a PHEP is in draft, ideas can come up which warrant further discussion. Those ideas should be recorded so people know that they are being thought about but do not have a concrete resolution. This helps make sure all issues required for the PHEP to be ready for consideration are complete and reduces people duplicating prior discussion.
+12. Footnotes - A collection of footnotes cited in the PHEP, and a place to list non-inline hyperlink targets.
+13. Copyright/license - Each new PHEP must be placed under a dual license of public domain and [CC0-1.0-Universal](https://choosealicense.com/licenses/cc0-1.0/) (see this PHEP for an example).
+
+# PHEP Formats and Templates
+PHEPs are UTF-8 encoded text files using the Markdown format. Markdown allows for rich markup that is still quite easy to read, but also results in good-looking and functional HTML. A future PHEP may include a template. PHEP Markdown must be compatible with the most recent version of the [CommonMark specification](https://spec.commonmark.org/) at the time the PHEP is approved. PHEPs must not use [GitHub extensions](https://github.github.com/gfm/).
+
+PHEPs should be restricted to the ASCII subset of UTF-8 where practicable. Examples of cases where it is not practicable include personal names.
+
+The PHEP text files are automatically rendered by the GitHub interface for easier online reading. The Markdown source is the version of record.
+
+The readability of the diff should be considered when editing a PHEP.
+
+# PHEP Header Preamble
+<a name="phep-header-preamble"></a>
+Each PHEP must begin with an [RFC 2822](https://datatracker.ietf.org/doc/html/rfc2822.html) style header preamble. The headers must appear in the following order. Headers marked with "*" are optional and are described below. All other headers are required.
+
+      PHEP: <phep number>
+      Title: <phep title>
+      Author: <list of authors' real names and optionally, email addrs>
+      Discussions-To: <URL of current canonical discussion thread>
+      Status: <Draft | Accepted | Rejected |
+               Withdrawn | Final | Superseded>
+      Type: <Standards Track | Informational | Process>
+      Content-Type: text/markdown; charset=UTF-8; variant=CommonMark
+    * Requires: <pep numbers>
+      Created: <date created on, in dd-mmm-yyyy format>
+      Post-History: <dates, in dd-mmm-yyyy format,
+                     inline-linked to PHEP discussion threads>
+    * Replaces: <pep number>
+    * Replaced-By: <pep number>
+    * Resolution: <url>
+
+The Author header lists the names, and optionally the email addresses of all the authors/owners of the PHEP. The format of the Author header values must be:
```

Comment:
Nit-picky comma use correction: `The Author header lists the names and, optionally, the email addresses of all the authors/owners of the PHEP.`